.github/RELEASE_CHECKLIST.md

```diff
@@ -1,24 +1,34 @@
 # Release checklist
+
 This checklist is for our own reference
 
-1. Check that everything is up to date and ready to go
-    - Travis tests are passing
-    - Manual tests on Bianca are passing
-2. Increase version numbers
-3. Update version numbers in code: `configuration/base.config`
-4. Build, and get the containers.
-    - `./scripts/do_all.sh --push --tag <VERSION>`
-    - `./scripts/do_all.sh --pull --tag <VERSION>`
-5. Test against sample data.
-    - Check for any command line errors
-    - Check version numbers are printed correctly
-    - `./scripts/test.sh -p docker --tag <VERSION>`
-    - `./scripts/test.sh -p singularity --tag <VERSION>`
-    - `./scripts/test.sh -p singularityPath --tag <VERSION>`
-6. Commit and push version updates
-7. Make a [release](https://github.com/SciLifeLab/Sarek/releases) on GitHub
-8. Choose an appropriate codename for the release
-9. Update [bio.tools](https://bio.tools/Sarek) with the new release
-10. Tweet that new version is released
-11. Commit and push. Continue making more awesome :metal:
-12. Have fika :cake:
+1.  Check that everything is ready to go
+
+    -   [PR](https://github.com/SciLifeLab/Sarek/pull) are merged
```

Comment:
Broken link I believe.  Also either PR:s or 'is'.

Reply:
Good point